chips/nrf52/src/lib.rs

```diff
@@ -1,6 +1,5 @@
-#![feature(
-    asm, concat_idents, const_fn, const_cell_new, try_from, core_intrinsics, iterator_step_by
-)]
+#![feature(asm, concat_idents, const_fn, const_cell_new, try_from)]
+#![feature(core_intrinsics, iterator_step_by, used)]
```

Comment:
Did you split this, or rustfmt? What determined what made the top list vs the bottom one?

Reply:
I did.